config-windows.md

```diff
@@ -0,0 +1,80 @@
+# Windows-specific Container Configuration
+
+This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
+
+
+#### Memory
+
+`memory` is used to set limits on the container's memory usage.
```

Comment:
No objection at all. Modified memory, CPU, storage, and networking to all match this pattern.